frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -233,6 +241,37 @@ describe('Cluster Version Details Table page while ClusterVersion Upgradeable=Fa
   });
 });
 
+describe('ClusterSettingsAlerts while cluster is up to date or has available updates and there are paused MachineConfigPools', () => {
+  let wrapper: ShallowWrapper<any>;
+  let cv;
+  let spyUseAccessReview;
+
+  beforeEach(() => {
+    spyUseAccessReview = jest.spyOn(utils, 'useAccessReview');
+    spyUseAccessReview.mockReturnValue(true);
+    cv = clusterVersionProps;
+    wrapper = shallow(
+      <ClusterSettingsAlerts
+        cv={cv}
+        machineConfigPools={machineConfigPoolsWithPausedWorkerProps.items}
+        status={ClusterUpdateStatus.UpToDate}
+      />,
+    );
+  });
+
+  afterEach(() => {
+    spyUseAccessReview.mockReset();
+  });
+
+  it('should render ClusterSettingsAlerts component', () => {
+    expect(wrapper.exists()).toBe(true);
```

Comment:
This will always be true...

See this issue for an official response about Enzyme: https://github.com/enzymejs/enzyme/issues/1864#issuecomment-430113911